less/index.less

```diff
@@ -52,8 +52,163 @@ table {
 * {
     box-sizing: border-box;
 }
+// div	{
+// 	border: 2px solid aqua;
+// }
 
-html, body {
-    height: 100%;
+body, html {
+    font-family: 'Josefin Sans', sans-serif; 
+    font-size: 62.5%;  
 }
+@y-space: 2.5%;
+@x-space: 15px;
+@color1: whitesmoke;
+@color2: lightgrey;
```

Comment:
Consider using more descriptive naming for your colors. Such as `primary-color`, `secondary-color` etc